From 92e2c5bfd6aba586e07c7e5a1e65848619bf8521 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Sun, 13 Oct 2013 16:02:19 +0200
Subject: [PATCH] Fix service path

Tweaks application is included by default now so it's not installed within /media/internal
anymore.

Upstream-Status: Inappropiate [configuration]

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 node-service/org.webosinternals.tweaks.prefs.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/node-service/org.webosinternals.tweaks.prefs.service b/node-service/org.webosinternals.tweaks.prefs.service
index 331938b..6bf27eb 100644
--- a/node-service/org.webosinternals.tweaks.prefs.service
+++ b/node-service/org.webosinternals.tweaks.prefs.service
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.webosinternals.tweaks.prefs
-Exec=/usr/bin/run-js-service -n /media/cryptofs/apps/usr/palm/services/org.webosinternals.tweaks.prefs
+Exec=/usr/bin/run-js-service -n /usr/palm/services/org.webosinternals.tweaks.prefs
-- 
1.8.1.2

